common/src/main/java/com/revenuecat/purchases/common/networking/Endpoint.kt

```diff
@@ -28,4 +28,17 @@ sealed class Endpoint(val pathTemplate: String, val name: String) {
         ) : Endpoint("/receipts/amazon/%s/%s", "get_amazon_receipt") {
         override fun getPath() = pathTemplate.format(Uri.encode(userId), receiptId)
     }
+
+    val supportsSignatureValidation: Boolean
+        get() = when (this) {
+            is GetCustomerInfo,
+            LogIn,
+            PostReceipt ->
+                true
+            is GetAmazonReceipt,
+            is GetOfferings,
+            is PostAttributes,
+            PostDiagnostics ->
```

Comment:
Any reason to repeat `is` in these? (I don't know enough Kotlin but it seems redundant).

Reply:
No,. I just forgot to clean it up 😅

Reply:
Ahh actually they are needed 😅 . The ones with `is` are data classes and we make sure the endpoint is an instance of that class. Those that don't have `is` are `objects`, and for those, only one can exist, so we don't need to check the type at all.